test/tests/phi_zero/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  [./phi_zero_linear]
+    type = 'RunApp'
+    input = 'simple_transient_diffusion.i'
+    expect_out = ' Solve Converged!'
+    issues = '#15204'
+    design = ' '
```

Comment:
ruh roh. You should either write some documentation or find some existing documentation that roughly fits what you're working on